README.md

```diff
@@ -17,309 +17,125 @@ Supports [CANNON.js](http://schteppe.github.io/cannon.js/) and [Ammo.js](https:/
 
-#### npm + webpack
+For more details, see detailed documentation for each driver 
 
-When using webpack, you need to ensure that your `loader` for `.js` files includes this dependency. The example below assumes you're using Babel.
+- [Cannon.js](CannonDriver.md#components)
 
-```js
-{
-  test: /\.js$/,
-  include: ['src', require.resolve('aframe-physics-system') ],
-  use: {
-    loader: 'babel-loader', // or whatever loader you're using to parse modules
-    options: {}
-  }
-}
-```
+- [Ammo.js](AmmoDriver.md#components)
 
-> **Note**: You cannot use `exclude: /node_modules` for your `.js` loader. You must instead use `include` and pass an array of directories as dependencies to transpile.
+## Constraints and APIs
 
-## Basics
+More sophisticated use cases require more than just the configuration of dynamic, static and kinematic bodies.
+
+Both drivers also allow for the configuration of constraints
+
+Constraints such as hinges, springs and so on can be configured between bodies (or between specific points on the surfaces bodies), to provide more sophisticated interactions.   See driver-specific documentation for details.
+
+Both drivers also have APIs that offer
+
+- lifecycle events such as a body initialization, entering a sleeping vs. active state etc.
+- collision events that can be used to detect collisions between bodies
+- direct interactions with bodies, for example setting their velocity, applying forces to them etc.
+
+Specific details  vary between drivers, so you should consult driver-specific documentation for details.  Since Cannon.js is written in native Javascript, its API is generally easier to use, and problems are simpler to debug.  In comparison, making use of the Ammo.js APIs can be quite hard work (there's definitely scope to improve the available documentation and examples here!)  
+
+- [Cannon.js](CannonDriver.md)
+
+- [Ammo.js](AmmoDriver.md)
+
+  
+
+## Driver-specific limitations
 
-```html
-<!-- The debug:true option creates a wireframe around each physics body. If you don't see a wireframe,
-     the physics system may be unable to parse your model without a shape:box or shape:hull option. -->
-<a-scene physics="debug: true">
+This is a list of limitations that has been oberved with particular drivers (and also with physx).  It's intended to provide a checklist to help developers to choose between physics drivers for a particular project, so they don't pick a driver that turns out to be missing some feature that is fundamental for their application.
 
-  <!-- Camera -->
-  <a-entity camera look-controls></a-entity>
+This list is probably incomplete, so if you find an additional significant limitation, please add it to this list.
 
-  <!-- Floor -->
-  <a-plane static-body></a-plane>
+### Cannon.js
 
-  <!-- Immovable box -->
-  <a-box static-body position="0 0.5 -5" width="3" height="1" depth="1"></a-box>
+- Can't handle collision with fast moving bodies, as it does not offer Continuous Collision Detection (CCD)
```

Comment:
I wrote my own CCD (no physics, but just for collision detection) recently, would be cool to figure out how to add this. Probably I should look upstream, first have been meaning to package the CCD as-is as a standalone package.

Reply:
of course, just a comment.